test/basic.js

```diff
@@ -51,6 +54,8 @@ models.forEach(function (model) {
 	uniqued[hash] = model;
 });
 
+console.time('perf');
```

Comment:
added a perf test, but no noticeable difference before and after this PR

(averages between 21ms and 26ms on my laptop)